content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,173 +2,304 @@
+   correct work entries for that employee. All conflicts for that employee are now resolved.
 
-.. image:: work_entries/conflicts.png
-   :align: center
-   :alt: Conflicts dashboard view showing all employee's conflicts in work entries.
+Generating payslips
+===================
 
-Conflicts are indicated with an orange triangle in the top left corner of each individual entry.
-Click on an individual work entry to see the conflict details in a pop-up.
+To generate payslips, :ref:`navigate to the time period <payroll/adjust-view>` the payslips should
+be generated for. Ensure the :guilabel:`Conflicting` filter is removed. When the desired pay period
+is displayed, click the :guilabel:`Generate Payslips` button.
 
-The conflict is briefly explained in an orange text box.
+.. tip::
+   If the :guilabel:`Generate Payslips` button is not active (appears pale purple instead of dark
+   purple), that indicates there are conflicts, or the date selected includes dates in the future.
+   Resolve all conflicts before generating payslips.
 
-.. image:: work_entries/conflict-detail.png
-   :align: center
-   :alt: Details for a conflict appear in the pop-up.
+A batch entry appears for the time period selected. The batch name populates the :guilabel:`Name`
+field in a default `From (date) to (date)` format.
 
-The :guilabel:`Work Entry Name`, :guilabel:`Employee`, and :guilabel:`Work Entry Type` are listed on
-the left side of the pop-up. The :guilabel:`From` and :guilabel:`To` date range, as well as the
-total time requested (in hours), appear in the :guilabel:`Period` field on the right side.
+The date range the payslips apply to appears in the :guilabel:`Period` field, and the company
+appears in the :guilabel:`Company` field. It is not possible to make changes to this form.
 
-If there is a conflict because a time-off request for the same time already exists in the system,
-the time-off will be entered in the :guilabel:`Time Off` field. Clicking the :guilabel:`External
-Link` button next to the :guilabel:`Time Off` entry will bring up the duplicate time-off request.
+Click the :guilabel:`Create Draft Entry` button to create the payslips for the batch.
 
-The details for the time-off request appear in the pop-up. The request can be modified if needed.
-Click either the :guilabel:`Validate` or :guilabel:`Refuse` button to approve or deny the request,
-then click the :guilabel:`Save` button to save the changes.
+Click the :guilabel:`Payslips` smart button in the top-right to view all the payslips for the batch.
 
-.. image:: work_entries/validate.png
+.. image:: work_entries/generate-payslips.png
    :align: center
-   :alt: Edit and/or validate a duplicate time-off request.
+   :alt: Information that appears when generating payslips.
+
+.. note::
+   Payslips have a status of *Waiting* until the :guilabel:`Create Draft Entry` button is clicked.
+   After, the payslip status changes to *Done*.
```

Comment:
In my runbot, and in your screenshot, I believe the statuses are: New, Confirmed, Done, and Paid. Can you confirm? If a change needs to be made, please adhere to the correct status names in the doc, if needed. Thanks 👍

Reply:
I am going to remove this, because you see this in the Payslips - All Payslips part of Payroll, not in the Work Entries. It seems like too much info here, and does not reflect the image. I was just worried people would do this part, then wonder why they say 'waiting' in the payslips section if they did what they thought they needed to here.